### PR TITLE
OBSDOCS-895: Create and publish release notes for the DT&OTEL 3.1.1 patch

### PR DESCRIPTION
**Pull request by max-cx**
- [OBSDOCS-694](https://issues.redhat.com//browse/OBSDOCS-694)
- [OBSDOCS-895](https://issues.redhat.com//browse/OBSDOCS-895): Create and publish release notes for the DT&OTEL 3.1.1 patch

The only new content is on these lines:

- https://github.com/openshift/openshift-docs/pull/73014/files#diff-8b557e0bad14cb628916d6113cdee0744c356268e0cc9d55d7e3c62d7b61225dR33-R36
- https://github.com/openshift/openshift-docs/pull/73014/files#diff-178dd58a084f23d38a7c15a25c97ca35c2a2fbd90071e6898260873fa1363498R11-R14

The rest of the "new" content in the diffs is already published content that was either copied and pasted or moved from one file o the other file:

- The file `distr-tracing-rn-3-1.adoc` is deleted and its content is moved to `distr-tracing-rn-past-releases.adoc`.
- The file `otel-rn-3.1.adoc` is deleted and its content is moved to `otel-rn-past-releases.adoc`.

Version(s):

4.16, 4.15, 4.14, 4.13, 4.12

Issue:

https://issues.redhat.com/browse/OBSDOCS-895

Link to docs preview:

Four pages in question (plus the edited OCP topic map file):

- https://73014--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1-1 (new page)
- https://73014--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases (edited page)
- https://73014--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel_rn/otel-rn-3-1-1 (new page)
- https://73014--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel_rn/otel-rn-past-releases (new page)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc [hunk 1]
```diff
@@ -8,6 +8,115 @@ toc::[]
...
+* Support for cluster-wide proxy environments.
+* Support for TraceQL to Gateway component.
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_bug-fixes"]
+==== Bug fixes
+
+This update introduces the following bug fixes for the {TempoShortName}:
+
+* Before this update, when a TempoStack instance was created with the `monitorTab` enabled in {product-title} 4.15, the required `tempo-redmetrics-cluster-monitoring-view` ClusterRoleBinding was not created. This update resolves the issue by fixing the Operator RBAC for the monitor tab when the Operator is deployed in an arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
+* Before this update, when a TempoStack instance was created on an {product-title} cluster with only an IPV6 networking stack, the compactor and ingestor pods ran in the `CrashLoopBackOff` state, resulting in multiple errors. This update provides support for IPv6 clusters.(link:https://issues.redhat.com/browse/TRACING-3226[TRACING-3226])
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc]: Fixed.

**Review by adellape — COMMENTED** [review]
LGTM. Recommend considering getting a redirect set up tho since `distr-tracing-rn-3-1.adoc` and `otel-rn-3.1.adoc` are going away.

**Review by IshwarKanse — APPROVED** [review]

**Comment by max-cx** [comment]
> LGTM. Recommend considering getting a redirect set up tho since `distr-tracing-rn-3-1.adoc` and `otel-rn-3.1.adoc` are going away.

FYI, the redirects are already requested.

**Review by pavolloffay — APPROVED** [review]

File: observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1-1.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+|{JaegerName} (deprecated)
+|Jaeger
+|1.53.0
+
+|===
+
+[id="distributed-tracing-rn_3-1-1_cves"]
+== CVEs
+
+This release fixes link:https://bugzilla.redhat.com/show_bug.cgi?id=2253330[CVE-2023-39326].
```
[the remark this answers is not in the document]
Comment (max-cx) [reply observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1-1.adoc]: Unfortunately, the Jira's security level is only for employees, so it wouldn't help linking to it. But I've just replaced the Bugzilla link with https://access.redhat.com/security/cve/cve-2023-39326, which I just learned about from @kcarmichael08 (thank you). Below are the links to the updated previews:

- https://73014--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1-1#distributed-tracing-rn_3-1-1_cves

- https://73014--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel_rn/otel-rn-3-1-1#otel-rn_3-1-1_cves

**Pull request merged**